current_fall2025/scripts/run_queries.py

```
@@ -0,0 +1,114 @@
+    embeddings = HuggingFaceEmbeddings(model_name="sentence-transformers/all-mpnet-base-v2")
+    llm = ChatOpenAI(model="gpt-4o-mini")
+
+    queries = load_queries(args.input)
+    results: List[Dict[str, Any]] = []
+
+    for row in queries:
+        qid, question = row["id"], row["question"]
+        t0 = time.time()
+        try:
+            answer, docs = RAG(llm, conn, embeddings, question, top=args.top, k=args.k)
+            logging.warning(f"[DEBUG] RAW ANSWER for {qid}: {repr(answer)}")
```

Ruff (0.14.8)

Use explicit conversion flag

Replace with conversion flag

(RUF010)